src/test/scala/uk/gov/nationalarchives/tdr/api/service/FileMetadataServiceSpec.scala

```diff
@@ -630,9 +606,8 @@ class FileMetadataServiceSpec extends AnyFlatSpec with MockitoSugar with Matcher
     FileMetadataFields.SHA256ServerSideChecksum should equal("SHA256ServerSideChecksum")
   }
 
-  private def generateFileStatusRows(fileIds: Seq[UUID], consignmentId: UUID = UUID.randomUUID()) = {
-    val timestamp: Timestamp = Timestamp.from(FixedTimeSource.now)
-    fileIds.map(id => FilestatusRow(consignmentId, id, "statusType", "value", timestamp))
+  private def generateFileStatusRows(fileIds: Seq[UUID]) = {
+    fileIds.map(id => FilestatusRow(null, id, "statusType", "value", null))
```

Comment:
I would still generate a status id and timestamp so the return object matches what would get returned by the DB call:

`... FilestatusRow(UUID.random(), id, "statusType", "value", Timestamp.from(FixedTimeSource.now)) ...`